pkg/network/tlsSecurityProfile.go

```diff
@@ -11,10 +11,27 @@ func SelectCipherSuitesAndMinTLSVersion(profile *ocpv1.TLSSecurityProfile) ([]st
 			Intermediate: &ocpv1.IntermediateTLSProfile{},
 		}
 	}
+	var ciphers []string
+	var minTlsVersion ocpv1.TLSProtocolVersion
 	if profile.Custom != nil {
-		return profile.Custom.TLSProfileSpec.Ciphers, profile.Custom.TLSProfileSpec.MinTLSVersion
+		ciphers = profile.Custom.TLSProfileSpec.Ciphers
+		minTlsVersion = profile.Custom.TLSProfileSpec.MinTLSVersion
+	} else {
+		ciphers = ocpv1.TLSProfiles[profile.Type].Ciphers
+		minTlsVersion = ocpv1.TLSProfiles[profile.Type].MinTLSVersion
 	}
-	return ocpv1.TLSProfiles[profile.Type].Ciphers, ocpv1.TLSProfiles[profile.Type].MinTLSVersion
+	// deduplicate ciphers
```

Comment:
no need comments please
they tend to age, and code is self explanatory here

Reply:
removed